test/common/ssl/context_impl_test.cc

```diff
@@ -246,8 +244,7 @@ TEST_F(SslContextImplTest, TestNoCert) {
   Json::ObjectSharedPtr loader = TestEnvironment::jsonLoadFromString("{}");
   ClientContextConfigImpl cfg(*loader, factory_context_);
   Runtime::MockLoader runtime;
-  Event::SimulatedTimeSystem time_system;
-  ContextManagerImpl manager(runtime, time_system);
+  ContextManagerImpl manager(runtime, time_system_);
```

Comment:
I guess while in here I would also put runtime and manager in the class to factor out these common initializations but this is pre-existing, so I will leave it up to you.

Reply:
I will do a follow-up PR with this